### ISSUE TITLE
Escape busy page?

### ISSUE THREAD

**Issue by BerkayTutal** [mgp25/Instagram-API#1034]
I'm trying to make a script which has more than one user but if one user tries to do something (ex:location search) the page blockes itself till receives a response from instagram.

I'm currently running it on local server but when it'll be online i'm going to receive the same error, happened me before because of using port 80 (ex:file uploading/downloading)

Probably I need async task or something similiar but I'm not a PHP expert.

So can anyone help me to solve this?

TL:DR : Page does not respond untill receiving response from Instagram.

**Comment by mgp25** [comment]
@BerkayTutal we only solve API issues

Regards
